### PR TITLE
test: add tests for empty token

### PR DESCRIPTION
**Pull request by sesky4**

### CODE REVIEW

File: src/test/java/com/tencentcloudapi/integration/requests/TempCredentialTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import java.util.Arrays;
+import java.util.List;
+
+public class TempCredentialTest {
+    @Test
+    public void TestEmptyToken() throws TencentCloudSDKException {
+        List<String> reqMethods = Arrays.asList("GET", "POST");
+        List<String> signMethods = Arrays.asList(
+                ClientProfile.SIGN_SHA1, ClientProfile.SIGN_SHA256, ClientProfile.SIGN_TC3_256);
+        List<String> tokens = Arrays.asList("abc", null);
```
Comment (zqfan) [inline src/test/java/com/tencentcloudapi/integration/requests/TempCredentialTest.java:19]: ""
